compiler/luci/import/src/Importer.cpp

```diff
@@ -51,10 +51,9 @@ void convert_graph(const luci::GraphBuilderSource &source, luci::CircleReader &r
   luci::GraphBuilderContext gb_context(graph, &reader, nodefinder.get(), tensoroutputs.get());
 
   const auto operators = reader.native_operators();
-  const auto &tensors = reader.tensors();
-  auto tensors_ptr = reader.tensors_ptr();
-  assert(tensors_ptr != nullptr);
+  const auto tensors = reader.native_tensors();
   auto circle_metadata = std::make_unique<luci::CircleImportMetadata>(reader);
+  assert(!tensors.null());
```

Comment:
maybe put this after `const auto tensors = reader.native_tensors();` line

Reply:
Done.